packages/EasyCodingStandard/packages/SniffRunner/tests/Application/FixerTest.php

```diff
@@ -56,6 +56,7 @@ public function testTokenContent(): void
     public function testAddContent(): void
     {
         $this->fixer->startFile($this->file);
+        $this->fixer->beginChangeSet();
```

Comment:
It must work without this call

Reply:
It must NOT work without this call. This is what's different in your Fixer and PHP_Codesniffer Fixer, behaviour I'm fixing.

Reply:
It MUST, since someone might forget it and it could lead to invalid states.

I'm open to handling original bad architecture of PHP_CodeSniffer, but not to replicate it.

Reply:
Not sure what you want from me. This test won't pass without this change, as it was incorrect since the start. Sniff can't forget this call and expect to add content to same token it already replaced.

Reply:
On second though, I this can be added to the code, since AFAIK only Symplify sniffs don't use it by default.

If you check + update Symlify sniffs (if these is any fixing sniff) with this, we can add it.

Reply:
I would be cautious in blindly adding them in your sniffs, without good understanding what are changesets used for. Most changes sniffs do are not done via changesets.

Reply:
I see. I don't remember using them - I switched to PHP CS Fixer as soon as I knew it - and no tests are failing. So let's consider this part accepted :+1: